example-get-started/generate.sh

```diff
@@ -135,6 +143,20 @@ dvc stage add -n train \
   -o model.pkl \
   python src/train.py data/features model.pkl
 dvc repro
+python <<EOF
+from dvc.repo import Repo
```

Comment:
feel we need a command for that? :)

may be it's easier append to dvc.yaml via bash?

Reply:
yes, we need that command I guess!

Let's use python approach if no strong objections.

Reply:
yep, no strong objections at all. Can use python, no worries. Thanks @aguschin for updating this repo.